src/configgen.py

```diff
@@ -17,6 +17,16 @@
 import re
 import textwrap
 from xml.dom import minidom, Node
+import io
+
+def xprint(str):
+    if sys.version_info[0] == 2:
+        str1 = unicode(str)
```

Comment:
I think this will create just an array of code-point mulitbyte characters, not UTF-8. I think you need to add `.encode('utf-8')` See also https://stackoverflow.com/a/10288345/784672

Reply:
This did work and I didn't see problems, I'll gave the `.encode('utf-8')` a try (again) and it gives the error:
```
[ 15%] Generating ../generated_src/configvalues.h
Traceback (most recent call last):
  File ".../doxygen/src/configgen.py", line 974, in <module>
    main()
  File ".../doxygen/src/configgen.py", line 788, in main
    xprint("/* WARNING: This file is generated!")
  File ".../doxygen/src/configgen.py", line 25, in xprint
    sys.stdout.write(str1)
TypeError: write() argument 1 must be unicode, not str

```